config/interface/hub/data.yml

```diff
@@ -146,6 +146,8 @@ Supporting:
     status: :prototype
     related_models:
       - Document 
+  ImportDataset:
+    status: :prototype
```

Comment:
I think it's more than prototype at this point.  It's had a year of testing?  It will be stable in its present form for a while...